joomla/joomla-cms
# Implement a database backup script [new feature]

### Summary of Changes
This patch adds a small database backup script to: Extensions => Manage => Database


### Testing Instructions
Go to Extensions => Manage => Database (make sure to have Super User rights) and click "Backup"

It's a POC. Feedback for improvement is welcome.
